lib/vm/src/instance/mod.rs

```diff
@@ -1000,9 +995,6 @@ impl InstanceHandle {
             VMBuiltinFunctionsArray::initialized(),
         );
 
-        // Ensure that our signal handlers are ready for action.
-        init_traps();
```

Comment:
The initialization has been moved in `Store`. Fine to me. I wonder what will happen if we have multiple stores, but that's because I'm ignorant of this code.

Another question: Is there a possible way to check whether the traps have been initialized? I think it's worth adding a check here to prevent against a bad usage of the API. Thought?
Edit: With [`Once::is_completed`](https://doc.rust-lang.org/std/sync/struct.Once.html#method.is_completed), it's possible to check whether the traps have been initialized correctly.

Reply:
`init_traps` is moved into the store. So it should be fine with multiple stores.

> Another question: Is there a possible way to check whether the traps have been initialized? I think it's worth adding a check here to prevent against a bad usage of the API.

The current API already prevents any misusage. Traps are initialized as soon as you init the store, so there should be no need for extra APIs or pushing the responsibility into the user